code/datums/components/flock/flockprotection.dm

```diff
@@ -36,8 +36,18 @@
 	if(!snitch)
 		return
 
+	var/report_name = source
+	if (istype(source, /mob/living/critter/flock/bit))
+		var/mob/M = source
+		report_name = M.real_name
+	else if (istype(source, /obj/flock_structure))
+		var/obj/flock_structure/structure = source
+		report_name = "the " + structure.flock_id
```

Comment:
byond has a text macro `\the` that prevents this from possibly becoming `the the collector`

Reply:
Likely needs to use string interpolation to get it to behave properly:
`"\the [structure.flock_id]"`

Reply:
Yeah I did try that